### ISSUE TITLE
Rewrite sorry/unsorry in python

### ISSUE THREAD

**Issue by jvperrin** [ocf/utils#121]
Currently [sorry](https://github.com/ocf/utils/blob/master/staff/acct/sorry/sorry) and [unsorry](https://github.com/ocf/utils/blob/master/staff/acct/sorry/unsorry) are pretty complicated scripts to be written in bash and have commented out sections, shellcheck-ignored warnings, etc. It's also tough to know if things done in the `sorry` script are undone properly in the `unsorry` script (setting file permissions for instance), so rewriting these two to use common functions and better error handling (maybe even be in the same script file) would be useful.

**Comment by gnowxilef** [comment]
oh wow, unsorry/sorry is still that terrible bash script? nice.

**Comment by jvperrin** [comment]
Yup, exactly, every time I have to look at it or use it and it breaks halfway through I get sad :'(
